src/zcl_abapgit_objects.clas.abap

```diff
@@ -821,35 +822,40 @@ CLASS ZCL_ABAPGIT_OBJECTS IMPLEMENTATION.
           lo_xml   TYPE REF TO zcl_abapgit_xml_output,
           lo_files TYPE REF TO zcl_abapgit_objects_files.
 
-    FIELD-SYMBOLS: <ls_file> LIKE LINE OF rt_files.
+    FIELD-SYMBOLS: <ls_file> LIKE LINE OF et_files.
 
-
-    IF is_supported( is_item ) = abap_false.
+    IF is_supported( cs_item ) = abap_false.
       IF NOT io_log IS INITIAL.
-        io_log->add( iv_msg = |Object type ignored, not supported: { is_item-obj_type
-                       }-{ is_item-obj_name }|
+        io_log->add( iv_msg = |Object type ignored, not supported: { cs_item-obj_type
+                       }-{ cs_item-obj_name }|
                      iv_type = 'E' ).
       ENDIF.
       RETURN.
     ENDIF.
 
     CREATE OBJECT lo_files
       EXPORTING
-        is_item = is_item.
+        is_item = cs_item.
 
-    li_obj = create_object( is_item = is_item
+    li_obj = create_object( is_item = cs_item
                             iv_language = iv_language ).
     li_obj->mo_files = lo_files.
     CREATE OBJECT lo_xml.
     li_obj->serialize( lo_xml ).
     lo_files->add_xml( io_xml      = lo_xml
                        is_metadata = li_obj->get_metadata( ) ).
 
-    rt_files = lo_files->get_files( ).
+    et_files = lo_files->get_files( ).
+
+    check_duplicates( et_files ).
 
-    check_duplicates( rt_files ).
+    TRY.
+        cs_item-inactive = boolc( li_obj->is_active( ) = abap_false ).
+      CATCH cx_sy_dyn_call_illegal_method.
```

Comment:
is this CATCH needed? `li_obj` has the right type

Reply:
was needed before all objects had is_active implemented => will remove it to cause a dump if some object implementation aren't adjusted

Reply:
now all objects has the method, so the CATCH can be removed?

Reply:
removed